spec/notification.js

```diff
@@ -63,12 +63,14 @@ describe('notification api', () => {
 
     });
 
-    it('should send an email with email_reply_to_id', () => {
+    it('should send an email with document upload', () => {
```

Comment:
Do we want one test for send email? Or another without the document upload?

Reply:
I will re-add the previous one, so there are two.